### PR TITLE
NDP References: Add FileDrop component

### PR DESCRIPTION
**Pull request by sorja**
- [x] Adds [dropzone](https://github.com/dropzone/dropzone) wrapper called `FileDrop`

### CODE REVIEW

File: src/client/components/FileDrop/FileDrop.scss [hunk 1]
```diff
@@ -0,0 +1,42 @@
+@import 'src/client/style/partials';
+
+$file-drop-border-radius: 4px;
+$file-drop-border-width: 2px;
+$file-drop-padding: 1rem;
+$file-drop-transition-time: 0.3s;
```
Comment (sorja) [inline src/client/components/FileDrop/FileDrop.scss:6]: Added internal variables, thoughts?
Looks more clean?

File: src/i18n/resources/en/ui.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+module.exports = {
```
Comment (sorja) [inline src/i18n/resources/en/ui.js]: Wasn't sure where to add, introduced new `ui` translation file for UI components

File: src/client/components/FileDrop/FileDrop.scss (hunk 1 above)
Comment (minotogna) [reply src/client/components/FileDrop/FileDrop.scss:6]: looks good!

File: src/client/components/FileDrop/FileDrop.scss [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  height: 100%;
+  padding: $file-drop-padding;
+
+  background-color: $ui-bg;
+  color: $text-body;
+  border-radius: $file-drop-border-radius;
+  border: $file-drop-border-width dashed $ui-border;
+
+  cursor: pointer;
+  transition: all $file-drop-transition-time ease-in-out;
```
Comment (minotogna) [inline src/client/components/FileDrop/FileDrop.scss:21]: I would remove new lines and order alphabetically

File: src/i18n/resources/en/ui.js (hunk 2 above)
Comment (minotogna) [reply src/i18n/resources/en/ui.js]: let's just have an entry fileDrop. ok ?

**Review by minotogna — APPROVED** [review]

**Pull request merged**